syntax = "proto3";

package chatroom.message;

message UserRegister {
    string name = 1;
    string password = 2;
}